### ISSUE TITLE
[sBTC DR] Should we clean Romeo state when possible

### ISSUE THREAD

**Issue by stjepangolemac** [stacks-network/sbtc#99]
When the deposits and withdrawals are fully processed, and all their transactions confirmed, should we remove them from Romeo state to keep it as small as possible — making it easier to understand it when debugging.

**NOTE**: Even if they're removed from the working state they'll still be included in the event log.

What do you think? @netrome @AshtonStephens @jferrant @sergey-shandar

**Comment by netrome** [comment]
Yep, I don't see why we wouldn't prune the running state. Especially since everything is visible from the event log.

**Comment by stjepangolemac** [comment]
Yeah I agree. The rule could be something like "delete successful sBTC operation data after N confirmation blocks", where N could be 256 or even 32. 🤷‍♂️

**Comment by stjepangolemac** [comment]
@friedger what do you think about this?

**Comment by friedger** [comment]
I think it is not the purpose of the signer to maintain a state for others. After work is done, we can forget..
